src/test/java/uk/gov/hmcts/ccd/v2/external/controller/CaseDocumentControllerIT.java

```diff
@@ -120,13 +69,8 @@ public void setUp() {
     @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_cases.sql"})
     public void shouldReturn200WhenGetValidCaseDocuments() throws Exception {
 
-        stubFor(com.github.tomakehurst.wiremock.client.WireMock.get(urlMatching("/api/data/case-type/TestCaseDocumentField"))
-                                                               .willReturn(aResponse()
-                                                                               .withHeader("Content-Type", "application/json")
-                                                                               .withBody(String.format(caseTypeResponseString, super.wiremockPort))));
-
```

Comment:
this and the above definition of 'caseTypeResponseString' seems to not affect the outcome of the test so I removed it.